cmd/token_test.go

```diff
@@ -72,7 +72,12 @@ func TestTokenWithGCEAK(t *testing.T) {
 			}
 			defer mockAttestationServer.Stop()
 
-			RootCmd.SetArgs([]string{"token", "--algo", op.algo, "--output", secretFile1, "--asAddr", mockAttestationServer.server.URL})
+			mockCloudLoggingServerAddress, err = newMockCloudLoggingServer()
```

Comment:
I would have expected you to check the `loggingHandler.logs` output matches up.

Reply:
The logs would need to contain a "pb.Attestation" struct from the TPM emulator. Do you have a reference or recommendation on how to create a deterministic "pb.Attestation"?

Reply:
The attestation would give different ak_cert since we are using tpm simulator.

Reply:
I create a new [PR](https://github.com/google/go-tpm-tools/pull/420) with the `loggingHandler.logs` output matches up. I use a simple field comparison for attestation.